kuma/health/views.py

```diff
@@ -156,3 +161,39 @@ def status(request):
     data['services']['test_accounts'] = test_account_data
 
     return JsonResponse(data)
+
+
+@csrf_exempt
+@require_POST
+def csp_violation_capture(request):
+    """
+    Capture CSP violation reports, forward to Sentry.
+
+    HT @glogiotatidis https://github.com/mozmeao/lumbergh/pull/180
+    HT @pmac, @jgmize https://github.com/mozilla/bedrock/pull/4335
+    """
+    if not settings.CSP_REPORT_ENABLE:
+        # mitigation option for a flood of violation reports
+        return HttpResponse()
+
+    data = client.get_data_from_request(request)
```

Comment:
I added a breakpoint here. The POST data, including the full report, is captured by ``client.get_data_from_request(request)``, and sent with ``client.captureMessage`` below.

Reply:
🎉 Awesome, thanks for checking that!